src/types/frame.ts

```diff
@@ -101,7 +101,7 @@ export type FrameResponse = {
    * @example
    * { width: 1200, height: 630 }
    */
-  imageOptions?: ImageOptions | undefined
+  imageOptions?: Omit<ImageOptions, 'fonts'> | undefined
```

Comment:
Let's slap a `Pretty` on there so the type gets evaluated and looks nicer.

```suggestion
  imageOptions?: Pretty<Omit<ImageOptions, 'fonts'>> | undefined
```

| Before | After |
|--------|--------|
| <img width="631" alt="image" src="https://github.com/wevm/frog/assets/6759464/8687fee3-3db2-4d3b-ab02-7cdad1df8dbb"> | <img width="676" alt="image" src="https://github.com/wevm/frog/assets/6759464/e1273cfd-2722-4cbf-a186-e2fbb9e9f071"> |

Reply:
holy shit, what a markdown art.